tests/FuzzTests.elm

```diff
@@ -43,6 +43,18 @@ constructing =
                         |> withDefault 42
                         |> expectZipper [] 42 []
             ]
+        , describe "#openAll"
+            [ fuzz (list int) "should focus on each element in the input (exactly once, in order)" <|
+                \l ->
+                    openAll l
+                        |> List.map (before >> List.length)
+                        |> Expect.equal (List.range 0 (List.length l - 1))
+            , fuzz (list int) "should not change the contents" <|
```

Comment:
Opted to check all the N^2 contents at once rather than just the current element. Again, stronger property.